packages/vm/test/retesteth/transition-child.cts

```diff
@@ -8,12 +8,12 @@ import { hexToBytes } from 'ethereum-cryptography/utils'
 import { readFileSync, writeFileSync } from 'fs'
 import { join } from 'path'
 
-import { VM } from '../../src'
-import { BlockBuilder } from '../../src/buildBlock'
+import { VM } from '../../dist/cjs'
```

Comment:
Should we import from src? (as we did before?)

Reply:
You know or may not matter since we are using `cts` to force it to be CommonJS. But let's be consistent one way or the other